easycv/apis/export.py

```diff
@@ -151,25 +152,61 @@ def _export_yolox(model, cfg, filename):
         cfg: Config object
         filename (str): filename to save exported models
     """
-    if hasattr(cfg, 'test_pipeline'):
-        # with last pipeline Collect
-        test_pipeline = cfg.test_pipeline
-        print(test_pipeline)
-    else:
-        print('test_pipeline not found, using default preprocessing!')
-        raise ValueError('export model config without test_pipeline')
+    cfg.model.type = 'YOLOXExport'
```

Comment:
it may be different for different model， we use
 yolox blade as name because  the blacklist op for blade is specialized for yolox, whether
useful for other cnn or transformer need to be checked later

Reply:
or support  export mode in model

Reply:
need fix